### DISCUSSION TITLE
Different color display for the response of ::assistant

### DISCUSSION THREAD

**Discussion by daniellsh** [bramses/chatgpt-md#18] [Ideas]
Currently, the two horizontal lines that differentiate ChatGPT's responses are quite clear. However, as the conversation continues to expand into lengthy pages, it would be visually beneficial to identify which response section belongs to ChatGPT by displaying the text in a different color.

**Comment by bramses** [comment]
That's a good idea, thank you @daniellsh. My main aim/dealbreaker rule with ChatGPT MD is to keep the boilerplate as minimal and as true to MD format as possible, but I can see how this would be useful, perhaps behind a setting. I'll look into it! Please feel free to open this as an issue so I can tag it!
